dev-tools/verify-fips.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Where is this actually called from?

Reply:
It's not called by anything, i'll remove to avoid bloat.